core/src/test/java/com/graphhopper/routing/RoundTripRoutingTest.java

```diff
@@ -48,7 +49,7 @@
  */
 public class RoundTripRoutingTest {
     private final FlagEncoder carFE = new CarFlagEncoder();
-    private final EncodingManager em = EncodingManager.create(carFE);
+    private final EncodingManager em = new EncodingManager.Builder().add(carFE).add(InSubnetwork.create("car")).build();
```

Comment:
I don't see why we would have to add insubnetwork for this test, it should work perfectly fine without anything related to subnetworks. It seems like we just add this here because `FiniteWeightFilter` requires the in subnetwork EV now (which I do not think it should).